pip/commands/download.py

```diff
@@ -63,6 +63,67 @@ def __init__(self, *args, **kw):
             help=("Download packages into <dir>."),
         )
 
+        cmd_opts.add_option(
+            '--platform',
+            dest='platform',
+            metavar='platform',
+            default=None,
+            help=("Only download wheels compatible with <platform>. "
+                  "Defaults to the platform of the local computer."),
+        )
+
+        cmd_opts.add_option(
+            '--interpreter-version',
```

Comment:
I'm not sur `interpreter-version` is the correct arg.
For example, latest PyPy interpreter version is 5.1.
It should maybe be `--python-version` ?